wallet/src/main/java/com/android/identity_credential/wallet/ui/prompt/consent/ConsentPromptEntryField.kt

```diff
@@ -296,10 +297,17 @@ private fun DataElementView(
 
         onClick = {},
         label = {
+            if (consentField.attribute?.icon != null) {
```

Comment:
How does this look if we're mixing both entries with icons and entries without them? Would it make sense to add a spacer if there's no icon, so the text is still aligned the same?

Reply:
It looks OK, actually, and it's an exception if the RP asks for a data element that isn't part of the profile. We might want to consider using a generic icon if we don't have one.